File: Animations/app/src/main/java/com/example/animations/MainActivity.java
package com.example.animations;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.provider.ContactsContract;
import android.util.Log;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        ImageView bartImageView = (ImageView) findViewById(R.id.imageView1);
        bartImageView.setX(-1000);
        bartImageView.animate().translationXBy(1000).rotation(3600).setDuration(2000);
    }

    boolean bartIsShowing = true;
    public void fadeBart(View view) {
        Log.i("Img","ImageView Tapped!");
        ImageView bartImageView = (ImageView) findViewById(R.id.imageView1);
        ImageView homerImageView = (ImageView) findViewById(R.id.imageView2);
        if(bartIsShowing) {
            bartIsShowing = false;
//            bartImageView.animate().alpha(0).setDuration(2000);
//            homerImageView.animate().alpha(1).setDuration(2000);
            bartImageView.animate().rotationBy(3600).scaleX(0f).scaleY(0f).setDuration(2000);
            homerImageView.animate().alpha(1).setDuration(2000);
        }
        else {
            bartIsShowing = true;
            homerImageView.animate().alpha(0).setDuration(2000);
//            bartImageView.animate().alpha(1).setDuration(2000);
            bartImageView.animate().rotationBy(3600).scaleX(1f).scaleY(1f).setDuration(2000);
        }
    }


}

File: Kotlin_View-Binding/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.EditText
import android.widget.TextView
import android.widget.Toast
import androidx.core.widget.addTextChangedListener

//import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val textView = findViewById<TextView>(R.id.textView)
        val editText = findViewById<EditText>(R.id.editText)
        val button = findViewById<Button>(R.id.button)
        with(textView){
            text = "Hello H3RMIT"
            setTextColor(getColor(R.color.design_default_color_primary_dark))
            textSize = 40f
        }
        // Or you can also use apply function
       editText.apply {
           hint = "Enter Your Name"
           setText("Hello There")
           addTextChangedListener {
               Log.i("TAG",it.toString())
           }
       }
       button.setOnClickListener {
           Toast.makeText(this@MainActivity,editText.text.toString(),Toast.LENGTH_SHORT).show()
       }
    }
}